package.json

```diff
@@ -35,6 +35,7 @@
     "ejs": "~2.3.4",
     "html-wiring": "~1.2.0",
     "mkdirp": "~0.5.1",
+    "underscore": "~1.8.3",
```

Comment:
Any reason to use underscore over lodash? Isn't lodash more performant and more maintained nowadays? I don't see this as an issue, but maybe just something to consider if changing it is just a matter of the dependency and using `_.assign` instead?